lib/project_types/rails/commands/create.rb

```diff
@@ -30,6 +30,9 @@ def call(args, _name)
         @ctx.abort(@ctx.message('rails.create.error.invalid_ruby_version')) unless
           Ruby.version(@ctx).satisfies?('~>2.5')
 
+        check_node
+        check_yarn
```

Comment:
I assume this is pretty much the same code that we run on the node side of things, apart from the messages. Should we centralize this somewhere so we don't have to maintain two versions of it?

Reply:
`check_node` is a duplicate but there's no `check_yarn` on the Node side.  Challenge with centralizing it is where?  In any case, we've got other duplicate code between rails and node that's somewhat intentional, due to the separation via the `project_types` directory structure.